node-w3capi/index.js

```diff
@@ -18,20 +18,66 @@
 // }
 
 import fs from "fs";
+import ora from "ora";
 import { fetchData } from "./api.js";
-import { renderHTML } from "./render.js";
+import { renderHTML } from "./html.js";
+import { renderMarkdown } from "./markdown.js";
 
 const SOLID_CG_ID = 110151;
 
-const { users, orgs } = await fetchData(SOLID_CG_ID);
+const filenames = {
+  json: {
+    users: "users.json",
+    orgs: "orgs.json"
+  },
+  html: {
+    participants: "participants.html"
+  },
+  md: {
+    affiliations: "affiliations.md"
+  }
+}
 
-fs.writeFileSync("users.json", JSON.stringify(users, null, 2));
-fs.writeFileSync("orgs.json", JSON.stringify(orgs, null, 2));
+const spinner = ora("🐌 Starting").start()
```

Comment:
I don't think we should add a package for a simple spinner which takes just a few lines of CSS.

Reply:
I haven't used CSS in CLI, no idea how that would work.

Reply:
I mean we can just have an HTML element and style it to spin?

Reply:
Oh nvm I am just noticing this library if for the CLI.. I thought it was in the HTML.